Update index.Rmd itn link and coursera phrasing

kweav opened the pull request:
Updating the itn link and coursera phrasing.

Can confirm that the feedback link takes you to the ITN form though the name may need updated in the form

Commit:
🔄 Synced local 'config_automation.yml' with remote 'config_automation.yml'

release-renderAction

Commit:
🔄 Synced local 'assets/style.css' with remote 'assets/style.css'

release-renderAction

Commit:
🔄 Synced local '.github/workflows/' with remote '.github/workflows/'

release-renderAction

Commit:
🔄 Synced local 'resources/exclude_files.txt' with remote 'resources/exclude_files.txt'

release-renderAction

Commit:
🔄 Created local 'assets/style_config_default.css' from remote 'assets/style_config_default.css'

release-renderAction

kweav committed:
Update index.Rmd itcr link and coursera phrasing

kweav committed:
Merge remote-tracking branch 'origin/repo-sync/OTTR_Template/default' into kweav-frontmatter-patch

kweav committed:
Update 05-promoting_diversity.Rmd to fix typo

kweav committed:
Update exclude_files.txt

Exclude a file with a lot of "typos" but not learner facing

kweav committed:
Update dictionary.txt

Add a word that is flagged but not a typo

carriewright11 commented:
Looks good - we can wait to merge this later :)

carriewright11 commented:
> Updating the itn link and coursera phrasing.
> 
> Can confirm that the feedback link takes you to the ITN form though the name may need updated in the form

the name matches the bookdown title... you mean that we need to consider the coursera title?

kweav commented:
> > Updating the itn link and coursera phrasing.
> > Can confirm that the feedback link takes you to the ITN form though the name may need updated in the form
> 
> the name matches the bookdown title... you mean that we need to consider the coursera title?

I made these comments about name mismatches before I had form access and updated the names in the form once I had access :) 

Aimed to have the bookdown title/Course Table names to match the names in the form rather than the github repo names which was mostly what it was before I updated

kweav commented on index.Rmd:
```diff
@@ -20,13 +20,13 @@ knitr::write_bib(c(
 - The material for this course can be viewed without login requirement on this [Bookdown website](https://jhudatascience.org/Informatics_Research_Leadership/). This format might be most appropriate for you if you rely on screen-reader technology.
 - This course can be taken for [free certification through Leanpub](https://leanpub.com/universities/courses/jhu/cancer-informatics-for-research-leaders).
-- This course can be taken on [Coursera for certification here](https://www.coursera.org/learn/leadership-for-cancer-informatics-research) (but it is not available for free on Coursera).
+- This course can be taken on [Coursera for certification for a fee here](https://www.coursera.org/learn/leadership-for-cancer-informatics-research) (this course can also be audited for free on Coursera).
```
```suggestion
- This course can be taken on [Coursera for certification for a fee here](https://www.coursera.org/learn/leadership-for-cancer-informatics-research).
```

carriewright11 replied:
looks good for when we merge for this course
